packages/launcher/src/renderer/launcher/IdentitySelectorView.js

```diff
@@ -103,17 +119,24 @@ export default class IdentitySelectorView extends Component<Props, State> {
       </View>
     ) : null
 
+    // TODO: scroll with max height
+
     return (
       <View>
         <Text style={styles.header}>{header}</Text>
-        {idRows}
         {createIdentity}
+        <ScrollView>
+          <View style={styles.scrollInner}>{idRows}</View>
```

Comment:
Couldn't this style be applied to the `ScrollView` itself? or maybe using the [`contentContainerStyle` prop](https://facebook.github.io/react-native/docs/scrollview#contentcontainerstyle) rather than nesting an extra `View`?